### PR TITLE
Adds format command

### PR DESCRIPTION
**Pull request by phelanjo**
Adds a !format command to help students format their code better on the Discord server.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ node_modules/
 .env
 build
 coverage/
+.prettierrc
```
Comment (ctsstc) [inline .gitignore]: @phelanjo Do you have a `.prettierrc` to add to the project?

**Comment by ctsstc** [comment]
An additional feature request could be to list all of the available languages, or even better link to a discord url if they have a page that lists the available languages. You may want to wrap the link in carrots so it doesn't display a preview like: `<https://some.link>`.

**Comment by ctsstc** [comment]
This has been pushed up to staging, you can try the command out by prefixing `;` and would recommend the `#testing` channel.

File: .gitignore (hunk 1 above)
Comment (phelanjo) [reply .gitignore]: I do! it's super basic though, just sets tab width and single quotes.

**Comment by phelanjo** [comment]
> An additional feature request could be to list all of the available languages, or even better link to a discord url if they have a page that lists the available languages. You may want to wrap the link in carrots so it doesn't display a preview like: `<https://some.link>`.

@ctsstc after some Googling, it looks like Discord doesn't have a page dedicated to the languages a user can use for code blocks, but they do use this library: https://highlightjs.org/static/demo/

**Pull request merged**